src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1004,14 +996,6 @@ def freeze_encoder(self):
         self.encoder._freeze_parameters()
 
     @add_start_docstrings_to_model_forward(WHISPER_INPUTS_DOCSTRING)
-    @add_code_sample_docstrings(
```

Comment:
This needs to be changed to use `replace_return_xxx` (forgot the exact name) instead.